vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/TouchActions.java

```diff
@@ -150,9 +157,23 @@ public void swipeUntil(SwipeDirection direction, Duration swipeDuration, Rectang
          * - payload is {"strategy": "accessibility id", "selector": "selector"}
          * - see https://appium.io/docs/en/commands/mobile-command/#android-uiautomator2-only
          * */
-
-        int screenshotHeightToCompare = genericWebDriverManager.getSize().getHeight() / HEIGHT_DIVIDER;
-        int heightOffset = screenshotHeightToCompare * 2;
+        SearchContext searchContext = uiContext.getOptionalSearchContext().orElse(null);
```

Comment:
I don't think we should do any actions if search context is not present (it means previous steps are failed)